src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1208,6 +1237,8 @@ private void UpdateEditor(float delta)
         {
             CurrentOrganelleCost = 0;
         }
+
+        editorArrow.Translation = editorArrow.Translation.LinearInterpolate(arrowPosition, ARROW_INTERPOLATE_SPEED);
```

Comment:
Probably not very necessary, but I think `ARROW_INTERPOLATE_SPEED` should be multiplied by `delta` so it won't be framerate dependent (as the arrow being near the object of attention, the speed difference may be very noticeable in low fps). May need to adjust the value to be at like `5.0f` to account for that.